docs/source/create_cla_docs.py

```diff
@@ -0,0 +1,25 @@
+from inspect import getmembers, isclass
+from genestack_client import cla
+
+
+template = '''
+{name}
+{name_underline}
+
+.. autoclass:: genestack_client.{name}
+        :members:
+        :show-inheritance:
+
+        .. autoattribute:: genestack_client.{name}.APPLICATION_ID
+
+'''
+
+import os
+print os.getcwd()
+
+
+def main():
+    with open('cla_reference._rst', 'w') as f:
```

Comment:
It was done to remove `WARNING: document isn't included in any toctree` but I found way to do it via `excludes` in `conf.py`